src/foam/nanos/notification/broadcast/BroadcastNotificationFacade.js

```diff
@@ -93,26 +166,58 @@ foam.CLASS({
         return ! errors_;
       },
       code: function() {
-        var a = this.BroadcastNotification.create({
+        var self = this;
+        var notif = this.BroadcastNotification.create({
+          template: this.template,
           body: this.body,
           toastMessage: this.toastMessage,
           groupId: this.groupId,
+          users: this.users,
+          emailArgs: this.emailArgs,
           severity: 'INFO',
           transient: false,
           toastState: this.showToast ? 'REQUESTED' : 'NONE'
         });
-        this.broadcastNotificationDAO.put(a).then(obj => {
-          this.notificationDAO.put(a).then(() => {
+        if ( this.users && this.users.length > 0 ) {
```

Comment:
What if users and group are selected, do we need to notify the user that its incorrect